pkgs/applications/misc/chrysalis/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, appimageTools, fetchurl }:
+
+let
+  pname = "chrysalis";
+  version = "0.7.9";
+in
+appimageTools.wrapType2 rec {
+  name = "${pname}-${version}-binary";
+
+  src = fetchurl {
+    url = "https://github.com/keyboardio/${pname}/releases/download/${pname}-${version}/${pname}-${version}.AppImage";
+    sha256 = "12w4vv7dwfpvxpc8kpfas90y7yy8mb8dj2096z3vw1bli5lrn3zi";
+  };
+
+  multiPkgs = null;
+  extraPkgs = p: (appimageTools.defaultFhsEnvArgs.multiPkgs p) ++ [
+    p.glib
+  ];
+
+  extraInstallCommands = "mv $out/bin/${name} $out/bin/${pname}";
```

Comment:
```suggestion
appimageTools.wrapType2 rec {
  pname = "chrysalis";
  version = "0.7.9";

  src = fetchurl {
    url = "https://github.com/keyboardio/${pname}/releases/download/${pname}-${version}/${pname}-${version}.AppImage";
    sha256 = "12w4vv7dwfpvxpc8kpfas90y7yy8mb8dj2096z3vw1bli5lrn3zi";
  };

  multiPkgs = null;
  extraPkgs = p: (appimageTools.defaultFhsEnvArgs.multiPkgs p) ++ [
    p.glib
  ];

  extraInstallCommands = "mv $out/bin/${name}-binary $out/bin/${pname}";
```

Didn't test.

Reply:
This actually doesn't work, as `appimageTools.wrapType2` isn't a (direct) wrapper around `stdenv.mkDerivation` but `buildFHSUserEnv` which then bails out with unknown arguments (`version`).